requirements.txt

```diff
@@ -1,5 +1,6 @@
+https://github.com/bottlepy/bottle/archive/7c1c1ca314f3eff8cc2deb894be5d02f7940fe3c.zip
```

Comment:
So this is current master, which is 0.13-dev, I suppose? Looks like we need it for the aiohttp support introduced in [884ef30](https://github.com/bottlepy/bottle/commit/884ef30e1b3fb891783cf56207d2f17f3645de59) (and possible fixes on top of that)?

Could you add a comment here?